### ISSUE TITLE
Run BBC-a11y against STY assets

### ISSUE THREAD

**Issue by chris-hinds** [bbc/simorgh#6222]
**Is your feature request related to a problem? Please describe.**
Currently we aren’t running bbc-a11y on CI/CD for any STY assets. This increases the chance we could accidentally make STY pages not accessible.

**Describe the solution you'd like**
Ensure that this tool is run on one STY and that any failures cause the build to fail.
The bbc-a11y module uses ./bbcA11y.js to configure its run. Add the STY page type here.

Note:
Investigate whether this test should also be run against test - if so, another issue should be raised to amend this file simorgh-infrastructure/config/articles.txt.

**Describe alternatives you've considered**

**Testing notes**
Run npm run dev && npm run bbcA11y to determine a11y tests are being run on STY assets.
Also verify it is being run correctly on CI and will cause the build to fail.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
https://github.com/bbc/bbc-a11y
